metacat-common-server/src/main/java/com/netflix/metacat/common/server/events/MetacatDeleteTablePostEvent.java

```diff
@@ -51,7 +52,27 @@ public MetacatDeleteTablePostEvent(
         @Nonnull @NonNull final Object source,
         @Nonnull @NonNull final TableDto table
     ) {
+        this(name, requestContext, source, table, false);
+    }
+
+    /**
+     * Constructor.
+     *
+     * @param name           name
+     * @param requestContext context
+     * @param source         The source object which threw this event
+     * @param table          table info
+     * @param isMView        true, if the table is a materialized view
+     */
+    public MetacatDeleteTablePostEvent(
+        @Nonnull @NonNull final QualifiedName name,
```

Comment:
I thought we decided not to do this with both the annotations? Don't we just need to add the annotation on the `package-info.java` and leave these annotations out completely unless we're putting `@Nullable` on